src/data.yaml

```diff
@@ -2908,6 +2908,19 @@ jobs:
         long: 
     review: 
     rating: 
+  - name: CarbonRe
```

Comment:
I think they should rather be listed under jobportals since they don't offer dev jobs themselves.

Reply:
They have done in the past but I think tech/dev jobs there are few and far between now they're more established.

- so sure thing I'll move to jobportals


I wonder,  ....
since it's not really a jobportal as such,
should they go in a new section that shows sites with lists of climate focussed companies
- their "our-portfolio" page would then be quite useful/appropriate

similarly this other one for example:
https://cambridgecleantech.org.uk/meet-our-members/

Reply:
I added the two list pages I had to a new section at the bottom of the Backlog until you decide if you want the new section or not

Carbon13's career's page (& jobs email list signup) - is in the Jobsportals section